app/code/Magento/InventoryBundleIndexer/Plugin/Indexer/SourceItem/BundleSourceItemIndexer.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\InventoryBundleIndexer\Plugin\Indexer\SourceItem;
+
+use Magento\InventoryBundleIndexer\Model\ResourceModel\BundleIndexer;
+use Magento\InventoryBundleIndexer\Model\ResourceModel\GetAllBundleSourceItemsIds;
+use Magento\InventoryBundleIndexer\Model\ResourceModel\GetBundleSourceItemsIds;
+
+class BundleSourceItemIndexer
```

Comment:
Provide clear name for plugin like
`Add bundle data to index`

Reply:
Done.